Automattic/wp-calypso
# Add spotlight banner on category pages

Fixes https://github.com/Automattic/dotcom-forge/issues/2788

## Proposed Changes

* add the spotlight block to plugins category pages
* set the message path for the page based on the logic in the `getMessagePathForJITM` selector

## Testing Instructions



* Patch D114133-code on the sandbox and proxy `public-api`
* Check the table in D114133-code test steps for which categories should return JITMs
* Visit `/plugins/:site` and browse categories where JITMs should be returned
* Check that JITMs are returned for these categories
* Browse other categories and check that JITMs aren't returned for these

| Category | Preview |
|--------|--------|
| Ecommerce & Business | ![image](https://github.com/Automattic/wp-calypso/assets/11555574/72a0fd00-906a-4fe1-9ae5-06156c4d61cd) |
| Search Engine Optimization | ![CleanShot 2023-06-22 at 15 44 36@2x](https://github.com/Automattic/wp-calypso/assets/11555574/37f5912a-857e-4a13-b492-9b5d55285380) | 

## Pre-merge Checklist
